ArchiSteamFarm/Steam/Bot.cs

```diff
@@ -178,11 +179,6 @@ public sealed class Bot : IAsyncDisposable, IDisposable {
 		}
 	}
 
-	/// <remarks>
-	///     Login keys are not guaranteed to be valid, we should use them only if we don't have full details available from the user
-	/// </remarks>
-	private bool ShouldUseLoginKeys => BotConfig.UseLoginKeys && (!BotConfig.IsSteamPasswordSet || !HasMobileAuthenticator);
```

Comment:
`BotConfig.UseLoginKeys` property is still available and should be either removed or renamed since "login keys" no longer exist. How about something more generic such as "SaveSession" or "StayLoggedIn". It could also be part of the `BotBehaviour` property. However, that one is currently only influences behaviour while `IsConnectedAndLoggedOn` is true and I'm not sure we want to mix things up.

Reply:
Rename is possible but I see little of value in it. The property will do exactly the same thing as before, but now we no longer need to work around broken login keys, which is even better.

It's a good poiunt, but I don't believe it's worth the effort.